src/js/tabs/firmware_flasher.js

```diff
@@ -141,225 +146,59 @@ firmware_flasher.initialize = function (callback) {
-                    if ($.inArray(target, unsortedTargets) === -1) {
-                        unsortedTargets.push(target);
-                    }
-                });
-                sortedTargets = unsortedTargets.sort();
-            });
-            sortedTargets.forEach(function(release) {
-                releases[release] = [];
-            });
-            releaseData.forEach(function(release) {
-                const versionFromTagExpression = /v?(.*)/;
-                const matchVersionFromTag = versionFromTagExpression.exec(release.tag_name);
-                const version = matchVersionFromTag[1];
-                release.assets.forEach(function(asset) {
-                    const targetFromFilenameExpression = /betaflight_([\d.]+)?_?(\w+)(\-.*)?\.(.*)/;
-                    const match = targetFromFilenameExpression.exec(asset.name);
-                    if ((!showDevReleases && release.prerelease) || !match) {
-                        return;
-                    }
-                    const target = match[2];
-                    const format = match[4];
-                    if (format !== 'hex') {
-                        return;
-                    }
-                    const date = new Date(release.published_at);
-                    const dayOfTheMonth = `0${date.getDate()}`.slice(-2);
-                    const month = `0${date.getMonth() + 1}`.slice(-2);
-                    const year = date.getFullYear();
-                    const hours = `0${date.getHours()}`.slice(-2);
-                    const minutes = `0${date.getMinutes()}`.slice(-2);
-                    const formattedDate = `${dayOfTheMonth}-${month}-${year} ${hours}:${minutes}`;
-                    const descriptor = {
-                        "releaseUrl": release.html_url,
-                        "name"      : version,
-                        "version"   : version,
-                        "url"       : asset.browser_download_url,
-                        "file"      : asset.name,
-                        "target"    : target,
-                        "date"      : formattedDate,
-                        "notes"     : release.body,
-                    };
-                    releases[target].push(descriptor);
-                });
+        function buildOptionsList(select_e, list) {
+            select_e.empty();
+            list.forEach(({ name, value }, index) => {
+                select_e.append($(`<option value='${value}'>${name}</option>`));
             });
-            loadUnifiedBuilds(releases);
         }
 
-        function supportsUnifiedTargets(version) {
-            return semver.gte(version.split(' ')[0], '4.1.0-RC1');
+        function buildOptions(data) {
+            buildOptionsList($('select[name="radioProtocols"]'), data.radioProtocols);
+            buildOptionsList($('select[name="telemetryProtocols"]'), data.telemetryProtocols);
+            buildOptionsList($('select[name="options"]'), data.generalOptions);
```

Comment:
Sorry almost forgot this one:

```suggestion
        function buildOptions(data) {
            if (navigator.connection) {
                buildOptionsList($('select[name="radioProtocols"]'), data.radioProtocols);
                buildOptionsList($('select[name="telemetryProtocols"]'), data.telemetryProtocols);
                buildOptionsList($('select[name="options"]'), data.generalOptions);
            }
```